python/amici/__init__.py

```diff
@@ -51,12 +55,14 @@
 amiciSrcPath = os.path.join(amici_path, 'src')
 amiciModulePath = os.path.dirname(__file__)
 
-from .sbml_import import SbmlImporter, assignmentRules2observables, \
-    constantSpeciesToParameters
-from .numpy import rdataToNumPyArrays, edataToNumPyArrays
-from .pandas import getEdataFromDataFrame, \
-    getDataObservablesAsDataFrame, getSimulationObservablesAsDataFrame, \
-    getSimulationStatesAsDataFrame, getResidualsAsDataFrame
+if has_clibs:
+    # these module require the swig interface
+    from .sbml_import import SbmlImporter, assignmentRules2observables, \
+        constantSpeciesToParameters
+    from .numpy import rdataToNumPyArrays, edataToNumPyArrays
+    from .pandas import getEdataFromDataFrame, \
+        getDataObservablesAsDataFrame, getSimulationObservablesAsDataFrame, \
+        getSimulationStatesAsDataFrame, getResidualsAsDataFrame
```

Comment:
looks reasonable, but hope it won't get more cases than clibs and hdf5 :)

Reply:
Yes, that's a bit messy. The problem is mostly, that for testing it is very helpful if you can import the package without requiring the compiled extensions. And the second problem is that there is no good way yet for having hdf5 available everywhere.